src/main/java/com/codeborne/selenide/impl/Plugins.java

```diff
@@ -42,7 +42,7 @@
     return implementation;
   }
 
-  private static <T> T getDefaultPlugin(Class<T> klass) {
+  public static <T> T getDefaultPlugin(Class<T> klass) {
```

Comment:
no need to make it public.

Reply:
need for my logic with loading default plugin for local run and custom plugin for remote

Reply:
reverted changes